code/modules/telesci/gps.dm

```diff
@@ -258,10 +257,61 @@ var/list/SPS_list = list()
 	var/alertarea = get_area(SPS)
 	var/alerttime = worldtime2text()
 	var/verbose = TRUE
+	var/boop = TRUE
 	var/transmission_data = "[alerttype] - [alerttime] - [alertarea] ([x0],[y0],[z0])"
 	for(var/obj/machinery/computer/security_alerts/receiver in security_alerts_computers)
 		if(receiver && !receiver.stat)
 			receiver.receive_alert(alerttype, transmission_data, verbose)
 			boop = TRUE
-	if (boop)
-		playsound(src,'sound/machines/radioboop.ogg',40,1)
+	
+	if(boop)
+		deathsound(isdead)
+
+/obj/item/device/gps/secure/proc/deathsound(var/dead=FALSE)
```

Comment:
you should just cut all the refs to `dead` since you're only calling this when the wearer dies anyway